packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api.py

```
@@ -168,6 +188,47 @@ def destroy_instance(self, instance: Instance) -> None:
+        Retrieve an instance's console id by its name.
+
+        Return None if it does not exist.
+        """
+        data = None
+
+        try:
+            res = self.req.get(f'{self.baseurl}/v1/instances/{instance.id}')
+            data = res.json()
+            res.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            msgerr = data.get('error') if data is not None else str(e)
```

Ruff (0.8.2)

Local variable `msgerr` is assigned to but never used

Remove assignment to unused variable `msgerr`

(F841)